gcn/notices/fermi/gbm/alert.schema.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/fermi/gbm/alert.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "GCN Schema for Fermi GBM Alert Notices",
+  "description": "This schema is used to report Fermi GBM Alerts.",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "../../core/AdditionalInfo.schema.json"
+    },
+    {
+      "$ref": "../../core/Alert.schema.json"
+    },
+    {
+      "$ref": "../../core/DateTime.schema.json"
+    },
+    {
+      "$ref": "../../core/Event.schema.json"
+    },
+    {
+      "$ref": "../../core/GeoLocBase.schema.json"
+    },
+    {
+      "$ref": "../../core/Reporter.schema.json"
+    },
+    {
+      "$ref": "../../core/Statistics.schema.json"
+    },
+    {
+      "$ref": "detectors.schema.json"
+    }
+  ],
+  "properties": {
+    "notice_type": {
+      "type": "string",
+      "description": "Notice title"
+    },
+    "hitl": {
+      "type": "boolean",
+      "description": "\"Human in the loop\".  When false, the notice was generated entirely automatically.  A value of true means that a human had an input, while the notice was being prepared.  The latter includes notices, which weren't altered by the human input, i.e. simply confirmed or approved by a human for further processing."
+    },
+    "trigger_algorithm_number": {
+      "type": "number",
+      "description": "The criterion number that caused the flight software to trigger. Not to be confused with flight software or any other version number."
```

Comment:
"Not to be confused with flight software or any other version number." - This doesn't really make sense.  Can you either list the corresponding energy range & timescale for that trigger_algorithm_number or refer to a table on the mission page that describes them?

Reply:
The trigger energy range and timescale are already included in the Alert notice.